jdi-light-html-tests/src/main/java/page/objects/selenium/HomePageEx.java

```diff
@@ -9,11 +9,17 @@
 
 
 public class HomePageEx extends PageBase {
-    public HomePageEx() { super("/index.html", "Home");}
-    @FindBy(css="form #name") public WebElement userName;
+    public HomePageEx() {
+        super("/index.html", "Home");
+    }
+
+    @FindBy(css = "#name")
+    public WebElement userName;
     public WebElement password = $("form #password");
 
-    @UI("form #name") public UIElement userNameUI;
+    @UI("#name")
+    public UIElement userNameUI;
     public UIElement passwordUI = $("form #password");
-    @UI("#login-button") public UIElement enterUI;
+    @UI("#login-button")
+    public UIElement enterUI;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818033)